MouseUnSnag.cs

```diff
@@ -364,6 +364,7 @@ public class MouseUnSnag
     private Point LastMouse = new Point (0, 0);
     IntPtr ThisModHandle = IntPtr.Zero;
     int NJumps = 0;
+    private bool wrap = true;

```

Comment:
You should call this `nowrap` and default this value to `false`...

```suggestion
    private bool nowrap = false;
```